spec/pluginmanager/commands/install_spec.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+require "spec_helper"
+require "pluginmanager/commands/install_command"
+require "uri"
+require 'webmock/rspec'
+
+describe LogStash::PluginManager::InstallCommand do
```

Comment:
Should the versions referenced in theses files uses the `LOGSTASH_VERSION`? They still mention 3.0.0-dev and master should be 5.0.0?